### ISSUE TITLE
Dangling ties

### ISSUE THREAD

**Issue by th-we** [music-encoding/sibmei#63]
How should dangling ties be encoded?  In any case, the current way of converting them in chords is flawed:

    <chord xml:id="m-46" dur="8" dur.ges="128p" stem.dir="down" visible="false">
        <note xml:id="m-47" dur="8" dur.ges="128p" oct="4" pname="g" pnum="67"/>
        <note xml:id="m-49" dur="8" dur.ges="128p" oct="4" pname="g" pnum="67"/>
    </chord>
    ...
    <tie xml:id="m-48" endid="#m-49" startid="#m-47"/>
    <tie xml:id="m-50" startid="#m-49"/>

I.e., the first tie goes from one chord note to the other, which does not make sense.  The second tie is maybe O.K., but I wonder whether it would be even more reasonable to encode it as laissez vibrer. Maybe `@lv` should be allowed on `<tie>` as well, but that's a topic for the main MEI issue tracker.

The following test case is taken from a "real life" document, but it's a little strange and probably an input error in this case:

[dangling-ties.zip](https://github.com/music-encoding/sibmei/files/750875/dangling-ties.zip)

**Comment by rettinghaus** [comment]
There are symbols for *laissez vibrer* available in Sibelius. However, they have to be positioned manually and are not exported. 
[LaissezVibrer.zip](https://github.com/music-encoding/sibmei/files/750951/LaissezVibrer.zip)

**Comment by lpugin** [comment]
Is `@tstamp` or `tstamp2` a possible solution?

**Comment by th-we** [comment]
@rettinghaus Yes, I know music fonts usually have dedicated symbols - the question in this case is more about how to handle situations where sibmei finds dangling ties.

Thinking about it: Danglign ties at the end of a repeat could actually have two intended endpoints: a note at the beginning of the repeat *and* a note in a second ending. Would we need multiple `@endid`s here?

**Comment by pe-ro** [comment]
`@tstamp2` can be used in combination with `@startid` for so-called "dangling ties" --
```
<tie startid="#note1" tstamp2="2.5"/>
```

**Comment by th-we** [comment]
@pe-ro That sounds like the solution to the problem!

**Comment by rettinghaus** [comment]
@th-we what is the status about this?

**Comment by th-we** [comment]
The ties no longer end on the wrong note, but the `@tstamp2` solution is not implemented yet.
